Sources/Jobs/JobQueueHandler.swift

```diff
@@ -68,6 +68,12 @@ final class JobQueueHandler<Queue: JobQueueDriver>: Service {
         var logger = logger
         let startTime = DispatchTime.now().uptimeNanoseconds
 
+        // Calculate wait time from queued to processing
+        Timer(
```

Comment:
This doesn't work. It assumes the machine running the job was the same as the machine that queued it. You need to use a value that has the same base eg 1 Jan 1970, not boot time.

Reply:
Exactly, this won't work if the job runs on another container instance or host

Reply:
Understood! The issue I faced is that, the swift date object is confusing to me. There's `timeIntervalSince1970`, `timeIntervalSinceNow`, `timeIntervalSinceReferenceDate` which one is available when or where is not clear.

Reply:
I updated the code to use Date.